### PR TITLE
Added logger to common module, usage examples witin GUI package

### PR DESCRIPTION
**Pull request by Tymoteuszauke**
Added LogUtils into common module of API.
Example usage is in ConnectController or BoardController classes.
Also fixed bug in fx/utils/PositionUtils

### CODE REVIEW

File: common/src/main/java/com/blackship/battlesheep/utils/LogUtils.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.blackship.battlesheep.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
```
Comment (mhbratek) [inline common/src/main/java/com/blackship/battlesheep/utils/LogUtils.java:6]: JavaDoc.

File: common/src/main/java/com/blackship/battlesheep/utils/LogUtils.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+public class LogUtils extends SecurityManager{
+
+    public static Logger getLogger() {
+        String className = new LogUtils().getClassName();
+        Logger logger = LoggerFactory.getLogger(className);
+        return logger;
+    }
+
+    private String getClassName()
+    {
```
Comment (mhbratek) [inline common/src/main/java/com/blackship/battlesheep/utils/LogUtils.java:18]: remove new line.

File: common/src/main/resources/log4j.properties [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Root logger option
```
Comment (mhbratek) [inline common/src/main/resources/log4j.properties:1]: :+1:

**Review by mhbratek — APPROVED** [review]
:beers:

File: client/src/main/java/com/blackship/battlesheep/fx/controllers/BoardController.java [hunk 4]
```diff
@@ -9,13 +10,15 @@
 import javafx.scene.control.TextArea;
 import javafx.scene.control.TextField;
 import javafx.scene.layout.GridPane;
+import org.slf4j.Logger;
```
Comment (arturAR) [inline client/src/main/java/com/blackship/battlesheep/fx/controllers/BoardController.java:13]: 👍

File: client/src/main/java/com/blackship/battlesheep/fx/controllers/utils/PositionUtils.java [hunk 5]
```diff
@@ -6,13 +6,13 @@
  */
 public class PositionUtils {
 
-    public static int calculateFromTwoDimensionalPosition(int columns, int row, int column) {
+    public static int calculateFromTwoDimensionalPosition(int columns, int column, int row) {
```
Comment (arturAR) [inline client/src/main/java/com/blackship/battlesheep/fx/controllers/utils/PositionUtils.java:9]: why? XD

File: common/src/main/java/com/blackship/battlesheep/utils/LogUtils.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by Mateusz on 30.07.2017.
+ */
+public class LogUtils extends SecurityManager{
+
+    public static Logger getLogger() {
+        String className = new LogUtils().getClassName();
```
Comment (arturAR) [inline common/src/main/java/com/blackship/battlesheep/utils/LogUtils.java:12]: 👍

**Review by arturAR — APPROVED** [review]

**Pull request merged**